.ado/jobs/sample-apps.yml

```
@@ -92,3 +92,46 @@ jobs:
             - template: ../templates/upload-build-logs.yml
               parameters:
                 buildLogDirectory: ${{ variables['BuildLogDirectory'] }}
+  - ${{ each config in parameters.buildMatrix }}:
+    - ${{ if eq(config.BuildEnvironment, parameters.buildEnvironment) }}:
+      - ${{ each matrix in config.Matrix }}:
+        - job: SampleAppFabric${{ matrix.Name }}
+          displayName: SampleApp Fabric ${{ matrix.Name }}
+
+          variables:
+            - template: ../variables/windows.yml
+            - name: BuildLogDirectory
+              value: $(Build.BinariesDirectory)\${{ matrix.BuildPlatform }}\${{ matrix.BuildConfiguration }}\BuildLogs
+
+          timeoutInMinutes: 60
+          cancelTimeoutInMinutes: 5
+          pool: ${{ parameters.AgentPool.Medium }}
+
+          steps:
+            - template: ../templates/checkout-shallow.yml
+
+            - template: ../templates/prepare-js-env.yml
+
+            - template: ../templates/prepare-build-env.yml
+              parameters:
+                platform: ${{ matrix.BuildPlatform }}
+                configuration: ${{ matrix.BuildConfiguration }}
+                buildEnvironment: ${{ config.BuildEnvironment }}
+
+            - template: ../templates/run-windows-with-certificates.yml
+              parameters:
+                buildEnvironment: ${{ parameters.BuildEnvironment }}
+                certificateName: sampleAppCPPEncodedKey
+                buildConfiguration: ${{ matrix.BuildConfiguration }}
+                buildPlatform: ${{ matrix.BuildPlatform }}
+                deployOption:  ${{ matrix.DeployOption }}
+                buildLogDirectory: ${{ variables['BuildLogDirectory'] }}
+                workingDirectory: packages/sample-app-fabric
+
+            - script: yarn bundle-cpp --verbose
```

```suggestion
            - script: yarn bundle --verbose
```

In order for this to work, you need to add the following to the "scripts" section of sample-app-fabric's `package.json`:

```json
"bundle": "rnw-scripts prepareBundle && react-native bundle --platform windows --entry-file index.windows.js --bundle-output windows/SampleAppFabric/Bundle/index.windows.bundle --assets-dest windows/SampleAppFabric/Bundle",
```